src/VfacTmdb/Results/Logo.php

```diff
@@ -0,0 +1,186 @@
+{
+    /**
+     * Id
+     * @var int
+     */
+    protected $id;
+    /**
+     * Id
+     * @var string
+     */
+    protected $logo_id;
+    /**
+     * Aspect ratio
+     * @var float
+     */
+    protected $aspect_ratio;
+    /**
+     * Logo file path
+     * @var string
+     */
+    protected $file_path;
+    /**
+     * Height
+     * @var int
+     */
+    protected $height;
+    /**
+     * Original file type
+     * @var string
+     */
+    protected $file_type;
+    /**
+     * Vote Average
+     * @var float
+     */
+    protected $vote_average;
+    /**
+     * Vote count
+     * @var int
+     */
+    protected $vote_count;
+    /**
+     * Width
+     * @var int
+     */
+    protected $width;
+
+    /**
+     * Constructor
+     * @param TmdbInterface $tmdb
+     * @param int $id
+     * @param \stdClass $result
+     */
+    public function __construct(TmdbInterface $tmdb, int $id, \stdClass $result)
+    {
+        $result->logo_id = $result->id;
+        $result->id      = $id;
+        parent::__construct($tmdb, $result);
+
+        $this->id           = $result->id;
+        $this->logo_id      = $result->logo_id;
+        $this->aspect_ratio = $result->aspect_ratio;
+        $this->file_path    = $result->file_path;
+        $this->height       = $result->height;
+        $this->file_type    = $result->file_type;
+        $this->vote_average = $result->vote_average;
+        $this->vote_count   = $result->vote_count;
+        $this->width        = $result->width;
+    }
+
+    /**
+     * Id
+     * @return int
+     */
+    public function getId() : int
+    {
+        return $this->id;
+    }
+
+    /**
+     * Aspect ratio
+     * @return float
+     */
+    public function getAspectRatio() : float
+    {
+        return $this->aspect_ratio;
+    }
+
+    /**
+     * Logo file path
+     * @return string
+     */
+    public function getFilePath() : string
+    {
+        return $this->file_path;
+    }
+
+    /**
+     * Original file type
+     * @return string
+     */
+    public function getFileType() : string
+    {
+        return $this->file_type;
+    }
+
+    /**
+     * Height
+     * @return int
+     */
+    public function getHeight() : int
+    {
+        return $this->height;
+    }
+
+    /**
+     * Id
+     * @return string
+     */
+    public function getLogoId() : string
+    {
+        return $this->logo_id;
+    }
+
+    /**
+     * Vote average
+     * @return float
+     */
+    public function getVoteAverage() : float
+    {
+        return $this->vote_average;
+    }
+
+    /**
+     * Vote count
+     * @return int
+     */
+    public function getVoteCount() : int
+    {
+        return $this->vote_count;
+    }
+
+    /**
+     * Width
+     * @return int
+     */
+    public function getWidth() : int
+    {
+        return $this->width;
+    }
+
+    /**
+     * Best logo file path
+     * @return string
+     */
+    public function getBestFilePath() : string
```

Comment:
It's a detail, but I'm not sure that this method name is the best. `getFilePath()` is more correct, we must not judge about the notion of "best" image. 
But we must comment this method to indicate the return choice of image what is done.

Reply:
The reason I used `getBestFilePath()` partially comes from the API docs:

> There are two image formats that are supported for networks, PNG's and SVG's. You can see which type the original file is by looking at the file_type field. We prefer SVG's as they are resolution independent and as such, the width and height are only there to reflect the original asset that was uploaded. 

The API seems to prefer SVGs and since the method prioritizes SVG over PNG it's returning the "best" image. Happy to rename it, but we already have a `getFilePath()` for the png filepath returned. I think that should always return the value as received from the API. I considered `getSvgPath()` but it won't always return an SVG (for some logos they do not exist). Maybe we add a parameter to the getter to request the SVG if available?

Reply:
Yes, I think that the parameter to the getter to request the SVG if available is a good idea. With `true` value by default if it's a boolean.